### PR TITLE
feat: Added Cart page

### PR DESCRIPTION
**Pull request by Devansu-Yadav**
- Added cart Management page
- Fixed active link on Side Menu
- Re-factored + added interactivity on empty Cart using JS

Preview link to the component - https://deploy-preview-5--bluekart.netlify.app/src/cart

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+}
+
+.cart-card-details {
+    height: 100%;
+    justify-content: flex-end;
+    flex-grow: 1;
+}
+
+.card-quantity-btns {
+    margin: 1rem 2rem 1rem;
```
Comment (codewithrishu) [inline css/cart.css:117]: you can try 1rem 2 rem also, bcz both top and bottom margin are 1 rem

**Review by codewithrishu — APPROVED** [review]
looks good to merge

File: assets/images/close-button.svg [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (anuj-kumary) [inline assets/images/close-button.svg]: Format this SVG file

**Review by anuj-kumary — APPROVED** [review]

**Review by lomesshh — APPROVED** [review]
good use of javascript to display empty cart
looks good to me

**Pull request merged**